### PR TITLE
Update stats explorer stats

### PR DESCRIPTION
**Pull request by Jibz1**
Screenshot 2023-01-20 at 11 48 32 AM

### CODE REVIEW

File: apps/explorer/src/components/validator/ValidatorStats.tsx [hunk 1]
```diff
@@ -94,11 +95,9 @@ export function ValidatorStats({
                                 {delegatedStakePercentage}%
                             </Heading>
                         </Stats>
-                        <Stats
-                            label="Self Staked"
-                            tooltip="Coming soon"
-                            unavailable
-                        />
+                        <Stats label="Self Staked" tooltip="Coming soon">
```
Comment (pchrysochoidis) [inline apps/explorer/src/components/validator/ValidatorStats.tsx:98]: Is tooltip correct now? (same for the other stats that now have values)

File: apps/explorer/src/components/validator/ValidatorStats.tsx [hunk 2]
```diff
@@ -116,14 +115,24 @@ export function ValidatorStats({
                         <Stats
                             label="Last Epoch"
                             tooltip="Coming soon"
-                            unavailable
-                        />
+                            unavailable={+lastEpoch <= 0}
```
Comment (pchrysochoidis) [inline apps/explorer/src/components/validator/ValidatorStats.tsx:118]: Can this be less than 0? And if last epoch was 0 why we don't show 0?

Comment (Jibz1) [reply apps/explorer/src/components/validator/ValidatorStats.tsx:118]: You are right, I'll remove it

**Review by mamos-mysten — APPROVED** [review]

**Comment by mystie711** [comment]
@Jibz1 went through all the tips in info hovers. May i request that for all tooltips that start with the word "The..." could we please remove it? It seems redundant. It will have the added benefit of some of the sentences not wrapping awkwardly to the next line with a lot of space left to the right.

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**